easybuild/easyconfigs/p/Python/Python-2.7.10-GCC-4.9.3-2.25-bare.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Python'
+version = '2.7.10'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
```

Comment:
For me, I'm happy to have most stuff at the Compiler level but some stuff like the visualisation libraries (wxWidgets only builds with GCC) are better off being done once properly and sitting in a common space (GCCcore). They bring with them quite a few dependencies though.

I wouldn't necessarily want to go to great lengths to populate GCCcore and I realise it's a bit painful to have to list binutils as a dep for everything built there...which I think you should do. My approach will be to pretty conservative about what I put there since it sits below the compiler level. Basically GCCcore is a dummy compiler that we control, if we wouldn't build it with dummy then we shouldn't build it with GCCcore.

Python in particular I was hoping to talk about during the user meeting to see how people would like to approach it. I think Python should be split between a base installation at compiler level and then extension bundles at MPI and MATH levels but I'm curious how others would approach it.